public/docs/features.md

```diff
@@ -1,3 +1,6 @@
+---
+tags: features, cool, updated
+---
```

Comment:
Are we sure to really tag the features page with these tags?
Maybe some instance users organize their notes for example with the "updated" tag and always have the features page mixed in-between.

Reply:
I know that, I just wanted to raise that question.

Reply:
Okay 2.0 thing.